### ISSUE TITLE
Make warping maps windows taller

### ISSUE THREAD

**Issue by timwaters** [wikimaps-dev/mapwarper#87]
see https://github.com/timwaters/mapwarper/issues/83

**Comment by susannaanas** [comment]
This should be solved in the new UI?

**Comment by timwaters** [comment]
yes... but this repository is for the warper and this issue is for the existing user interface. And therefore it should remain open until either fixed or replaced.
